schema/spacewalk/common/tables/suseContentEnvironmentTarget.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Copyright (c) 2018 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseContentEnvironmentTarget(
+    id NUMBER NOT NULL
+        CONSTRAINT suse_ct_env_tgt_id_pk PRIMARY KEY,
+
+    env_id NUMBER
+        CONSTRAINT suse_ct_env_tgt_eid_fk
+            REFERENCES suseContentEnvironment(id)
+            ON DELETE CASCADE,
+
+    type VARCHAR2(16) NOT NULL,
+
+    channel_id NUMBER
+        CONSTRAINT suse_ct_env_tgt_chanid_fk
+            REFERENCES rhnChannel(id)
+            ON DELETE CASCADE,
+
+    config_channel_id NUMBER
+        CONSTRAINT suse_ct_env_tgt_conchanid_fk
+            REFERENCES rhnConfigChannel(id)
+            ON DELETE CASCADE
```

Comment:
I think we do not want to implement "Config Channel" in the first iteration. We just used it as example to not forget about it.
I think it should be not in the code or DB schema yet.

Reply:
Ok, let's simplify that.